lib/content-services/document-list/components/document-list.component.ts

```diff
@@ -618,34 +616,12 @@ export class DocumentListComponent implements OnInit, OnChanges, OnDestroy, Afte
             this.setupDefaultColumns(this._currentFolderId);
         }
 
-        this.loadFolderByNodeId(this._currentFolderId);
-    }
-
-    loadFolderByNodeId(nodeId: string) {
-        if (this.customResourcesService.isCustomSource(nodeId)) {
-            this.updateCustomSourceData(nodeId);
-            this.customResourcesService.loadFolderByNodeId(nodeId, this._pagination, this.includeFields)
-                .subscribe((nodePaging: NodePaging) => {
-                    this.onPageLoaded(nodePaging);
-                }, (err) => {
-                    this.error.emit(err);
-                });
-        } else {
-
-            this.documentListService.getFolder(null, {
-                maxItems: this._pagination.maxItems,
-                skipCount: this._pagination.skipCount,
-                rootFolderId: nodeId,
-                where: this.where
-            }, this.includeFields)
-                .subscribe((nodePaging: NodePaging) => {
-                    this.getSourceNodeWithPath(nodeId).subscribe((nodeEntry: NodeEntry) => {
```

Comment:
getSourceNodeWithPath is disappeared on your PR we need it otherwise the breadcrumb and other functionality will not work

Reply:
fixed